test/e2e/tsconfig.json

```
@@ -2,10 +2,10 @@
 	"$schema": "https://json.schemastore.org/tsconfig.json",
 	"extends": "../../tsconfig.base.json",
 	"compilerOptions": {
+		"checkJs": false,
 		"noEmit": true,
-		"emitDeclarationOnly": false,
-		"allowJs": true,
-		"checkJs": false
+		"rootDir": ".",
+		"types": [ "node" ]
 	},
 	"include": [ "**/*" ],
```

We should be able to simplify this glob:
```suggestion
	"include": [ "." ],
```

---

Done. Thanks